config/config.go

```diff
@@ -350,6 +353,8 @@ func NewConfig() *Config {
 	flag.StringVar(&cfg.InnkeeperPreRouteFilters, "innkeeper-pre-route-filters", "", "filters to be prepended to each route loaded from Innkeeper")
 	flag.StringVar(&cfg.InnkeeperPostRouteFilters, "innkeeper-post-route-filters", "", "filters to be appended to each route loaded from Innkeeper")
 	flag.StringVar(&cfg.RoutesFile, "routes-file", "", "file containing route definitions")
+	flag.Var(cfg.RemoteRoutesURLs, "remote-routes-urls", "comma separated URLs to route definitions in eskip format")
+	flag.BoolVar(&cfg.RemoteRoutesFailOnStartup, "remote-routes-fail-on-startup", false, "fail fast when an initial attempt of fetching remote routes is unsuccessful")
```

Comment:
I think it should be `true` by default like for the files (fails if route file is malformed).

Reply:
The thing is the error in question is not about parsing routes in this case, but availability of the remote endpoint.